crates/zksync/core/src/vm/runner.rs

```diff
@@ -51,7 +51,10 @@ where
     let caller = env.tx.caller;
     let nonce = ZKVMData::new(&mut ecx).get_tx_nonce(caller);
     let (transact_to, is_create) = match env.tx.transact_to {
-        TransactTo::Call(to) => (to.to_h160(), false),
+        TransactTo::Call(to) => {
```

Comment:
had to change this one since transact may get deployments of the libs now, and we need to have the outcome parsed appropriately
technically speaking this is also helpful for broadcastable txs, as those could also contain deployments in this form